frontend/controller/actions/chatroom.js

```diff
@@ -42,7 +44,7 @@ sbp('okTurtles.events/on', MESSAGE_RECEIVE_RAW, ({
     messageHash: msgData.hash,
     height: msgData.height,
     text: msgData.text,
-    isDMOrMention: isMentionedMe || getters.chatRoomAttributes.type === CHATROOM_TYPES.DIRECT_MESSAGE,
+    isDMOrMention: isMentionedMe || targetChatroomState.attributes.type === CHATROOM_TYPES.DIRECT_MESSAGE,
```

Comment:
This was the cause of the issue.
`getters.chatRoomAttributes` getter here is for a currently active chatroom which means the DM notification only works when the receivers of the DM have the chatroom open in their browsers and this is what I observed too.

Made a fix here where it checks if the target chatroom is DM or not, regardless of whether the chatroom is active or not.